### ISSUE TITLE
pybamm.erf and pybamm.erfc are missing documentation

### ISSUE THREAD

**Issue by partben** [pybamm-team/PyBaMM#1623]
Both pybamm.erf and pybamm.erfc are entirely missing from the documentation, but are present in the latest version of PyBaMM.

**Comment by brosaplanella** [comment]
Actually, there are other functions missing too (e.g. `ArcTan`, `ArcSinh`, `Sqrt`) so it would be good to include all of them in the docs. It would also be good to reorganise `functions.py` so the functions appear in alphabetical order.

**Comment by tinosulzer** [comment]
Yes should be easy to fix, just add the missing pieces to this file https://github.com/pybamm-team/PyBaMM/blob/develop/docs/source/expression_tree/functions.rst
Can you do this @partben ?
